test/specs/repo/extensions/babel.ts

```diff
@@ -1,18 +1,16 @@
 import assert from 'assert';
 import LoginPage from 'wdio-mediawiki/LoginPage.js';
-import { skipIfExtensionNotPresent } from '../../../helpers/default-functions.js';
-
```

Comment:
Spacing

Reply:
Good catch. This is now resolved but for some reason Github didn't catch that.